praw/objects.py

```diff
@@ -148,7 +148,9 @@ def _get_json_dict(self):
             not wiki_page and scope('read')
 
         try:
-            params = {'uniq': self._uniq} if self._uniq else {}
+            params = self._params
+            if self._uniq:
```

Comment:
Question, why are we using `self._params`? Maybe I'm just missing something obvious that I'm misunderstanding.